modules/fts/pages/fts-creating-full-text-aliases.adoc

```diff
@@ -24,4 +24,12 @@ In the Add Alias page, add the alias name in the *Index Name* field. After that
 Finally, click *Create Index Alias*. The new index alias is added to the list in the Full Text Aliases panel.
 
 [#fts_add_alias_page_with_alias]
-image::fts-full-text-aliases-page-with-alias.png[,700,align=left]
\ No newline at end of file
+image::fts-full-text-aliases-page-with-alias.png[,700,align=left]
+
+=== **R**ole **B**ased **A**ccess **C**ontrol rules for Index-Alias
+
+For accessing an index-alias, the user needs to have the reader permissions for all the comprising index definitions in the alias.
+
+=== Index Lifecycle Implications
+
+If any of the index definition comprised within the index-alias gets deleted, then the index-alias would become invalid immediately. It would throw respective error messages upon any further access.
```

Comment:
Should `invalid` be `ineligible` instead ?

Reply:
IMHO invalid reads just fine, as in we are invalidating that definition.